code/game/machinery/machinery.dm

```diff
@@ -242,7 +242,7 @@ Class Procs:
 		return 1
 	if(usr.is_mob_restrained() || usr.lying || usr.stat)
 		return 1
-	if (!ishuman(usr))
+	if (!usr.IsAdvancedToolUser() && !isxeno(usr))
```

Comment:
```suggestion
	if (!usr.IsAdvancedToolUser())
```
The check should be `isxenoqueen`, but restricted to the dropship.

Reply:
Xenos aren't advanced tool users, as far as I know. Should I change it?

Reply:
The old version was basically `if(!ishuman(usr) && !ismonkey(usr) && !issilicon(usr) && !isxeno(usr))`.
computers usually don't return `attack_hand` on `attack_alien`, with the exception of these dropship ones.